loginas/utils.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from django.conf import settings as django_settings
+from django.contrib.auth.models import User
+from django.contrib.auth import load_backend, login, logout
+from django.contrib import messages
+from django.core.signing import TimestampSigner, SignatureExpired
+from datetime import timedelta
+
+from . import settings
```

Comment:
Oh oops, I didn't notice that you were importing settings there, sorry about that. In that case, since `settings` actually does contain all the Django settings, I'm fine with it if you want to leave it (or change it back, sorry!).